.github/workflows/release.yml

```diff
@@ -289,6 +289,28 @@ jobs:
           asset_name: saucetpl.tar.gz
           asset_content_type: application/tar+gzip
 
+  # Post Deploy Tests
+  post-release-windows-tests:
+    runs-on: ubuntu-latest
+    needs: release-windows-bundle
+    env:
+      SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+      SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
+    steps:
+      - uses: actions/checkout@master
+      - name: Install Deps
+        run: npm ci
+      - name: saucectl test
+        working-directory: ./tests/fixtures/devxpress-test
+        # TODO: Add runner_version as a parameter so that it can pull from github releases
+        run: |
+          VERSION=$(curl -s -H "Authorization: token ${{ github.token }}" \
+            https://api.github.com/repos/${{ github.repository }}/releases | \
+            jq -r "[.[] | select(.draft == true) | select(.body | contains(\"- jobId: ${{ github.run_id }}\\n\"))] | first | .tag_name")
+          echo Version:
+          echo $VERSION
+          npx saucectl run --test-env sauce --ccy 5 --region us-west-1 --runner-version "github-release: $VERSION"
+
   publish-release:
     runs-on: ubuntu-latest
     needs: [release-docker,  release-template-bundle, release-windows-bundle, release-macos-bundle]
```

Comment:
`release-windows-bundle` should be added here, except if we want to publish the release in case of failure.

Reply:
* saucectl is moved to devDependencies so that it can be run in pipelines but it won't be included in the runner bundles
* "release-windows-bundle" is there already in `needs:` or am I misunderstanding something?

Reply:
@dpgraham my bad, bad copy/paste, I was meaning the `post-release-windows-tests`